app/pages/ProjectsPage.tsx

```diff
@@ -45,11 +46,23 @@ ProjectsPage.loader = async () => {
   return null
 }
 
+const colHelper = createColumnHelper<Project>()
+const projectsPageStaticCols = [
```

Comment:
Nitpick, but for future reference I think the long name is overkill in files where there's only one, which is nearly all of them. It's actually kind of nice for them all to have the same name; it's like `export const loader` in a remix app.